tests/e2e/workloads/test_create_resources_using_kube_job.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from time import sleep
+
+from ocs_ci.ocs.cluster import ceph_health_check
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    tier1,
+    ManageTest,
+    ignore_leftovers,
+)
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_aws_i3,
+    skipif_bm,
+    skipif_external_mode,
+    skipif_ibm_cloud,
+    skipif_ibm_power,
+    skipif_lso,
+    ipi_deployment_required,
+    managed_service_required,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@ignore_leftovers
+class TestCreateResourcesUsingKubeJob(ManageTest):
+    """
+    Test create resources using the kube job
+    """
+
+    @skipif_aws_i3
+    @skipif_bm
+    @skipif_lso
+    @skipif_ibm_cloud
+    @skipif_ibm_power
+    @skipif_external_mode
+    @ipi_deployment_required
+    def test_create_resources_using_kube_job(self, create_resources_using_kube_job):
```

Comment:
I dont see any managed services marker to this test, so I assume that this is a non MS test.
If that is the case, why the test has to skip on upi, external, bm, lso etc..

Reply:
I saw here https://github.com/red-hat-storage/ocs-ci/blob/73cfafd0f98ca485fda388f6e9cf443d87788c2f/ocs_ci/ocs/scale_lib.py#L271 that we use just specific platforms and deployment types. That's why I added all the skips above. Maybe we can remove some of them.

Reply:
Done